pkgs/modules/python/default.nix

```diff
@@ -113,6 +113,7 @@ in
     python3-wrapper
     pip-wrapper
     poetry-wrapper
+    pkgs.uv
```

Comment:
Do we need the python wrapper for uv like we're doing for pip and poetry?

Reply:
I don't believe so. I'm still doing manual testing, but `uv` relies so little on the environment there's _probably_ not much use

Reply:
great!